Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfilePictureController.cs

```diff
@@ -57,6 +61,12 @@ public async Task<ActionResult> Post()
                 return BadRequest();
             }
 
+            if (file.Length > _maxFileSize)
+            {
+                _logger.LogError("File give to image was too large");
+                return BadRequest();
+            }
```

Comment:
asp.net core has a built-in filter for this type of thing. there's a default request size limit of 30 MB or so (though it's been disabled on this method), and if you want to lower that, then look up RequestSizeLimitAttribute.

but  this should have been done with TDD into this branch. so start over with this change - add failing test(s), then make the tests pass in a separate PR into the feature branch.